FeatureDetect/src/samples/test.cpp

```diff
@@ -0,0 +1,7 @@
+#include "featurelib.h"
+
+int main()
+{
```

Comment:
Let's pick a codestyle that is supported by some linter and stick to it.

For example, if we are writing for Android mainly, we can use [this one](https://google.github.io/styleguide/cppguide.html) from Google. It is supported in [cpplint](https://github.com/cpplint/cpplint).

Reply:
i install that, done